### PR TITLE
feat: add new javascript quiz

### PR DESCRIPTION
**Pull request by Evavic44**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

File: src/data/javascript-quiz.ts [hunk 1]
```diff
@@ -1509,6 +1411,17 @@ const javascriptQuiz = [
     Explanation: "?. is the optional chaining operator in JavaScript.",
     Link: "https://www.freecodecamp.org/news/javascript-optional-chaining/",
   },
+  {
+    Question:
+      "What is the result of the following: ('b' + 'a' + + 'cteri' + 'a').toLowerCase()",
```
Comment (jdwilkin4) [inline src/data/javascript-quiz.ts:1416]:
```suggestion
      "In JavaScript, what is the result for the following code: ('b' + 'a' + + 'cteri' + 'a').toLowerCase()?",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Review by Evavic44 — COMMENTED** [review]
Hey, @jdwilkin4. I've made the recommended changes. 🙂

**Review by jdwilkin4 — APPROVED** [review]
Congrats on your first contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**